sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/BeamSqlEnv.java

```diff
@@ -158,7 +90,7 @@ public void executeDdl(String sqlStatement) throws ParseException {
     try {
       SqlExecutableStatement ddl = (SqlExecutableStatement) planner.parse(sqlStatement);
       ddl.execute(getContext());
-    } catch (SqlParseException e) {
+    } catch (Exception e) {
       throw new ParseException("Unable to parse DDL statement", e);
```

Comment:
I think it will be cleaner to wrap the `try/catch` inside of `CalciteQueryPlanner`, this way you keep planner-specific exceptions

Reply:
I might change it back to what it originally is. When adding nay new planner, we can have BeamSQL general/planner specific exception, unless there is better option now.

Reply:
If you wrap specific exceptions in the implementation of `QueryPlanner` you don't need to care here